edge-util/src/Microsoft.Azure.Devices.Edge.Util/module/ModuleUtil.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Util.module
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Client.Transport.Mqtt;
+    using Microsoft.Azure.Devices.Edge.Util.TransientFaultHandling;
+    using Microsoft.Extensions.Logging;
+    using Serilog;
+    using Serilog.Core;
+    using Serilog.Events;
+    using ExponentialBackoff = Microsoft.Azure.Devices.Edge.Util.TransientFaultHandling.ExponentialBackoff;
+    using ILogger = Microsoft.Extensions.Logging.ILogger;
+
+    public static class ModuleUtil
+    {
+        public static readonly ITransientErrorDetectionStrategy DefaultTimeoutErrorDetectionStrategy =
```

Comment:
It is probably okay for now.. but I am a little concerned about Util becoming a dumping ground for all common code. 
Maybe we need to refactor this slightly better..

Reply:
Got it. Create a new project ModuleLib; since the intent is just to share code between modules; and don't make dirty in Edge.Util project.